src/components/Dashboard.jsx

```diff
@@ -1,40 +1,26 @@
 import React, {Component} from 'react';
 import {Route, Switch} from 'react-router-dom';
+import {connect} from "react-redux";
 
 import DashboardHeader from './DashboardHeader';
 import DashboardSiderbar from './DashboardSiderbar';
 import DashboardMain from './DashboardMain';
 import ProfileRouter from './ProfileRouter';
-import client from '../api/apiInstance';
-import {basicHeaders} from '../config/apiConfig';
 import ManagementPanel from './ManagementPanel';
-import {unsafeApiError} from '../utilities/utilities';
+import {validateLogin} from '../redux/actions/authActions';
 
 import '../stylesheets/dashboard.scss';
 
 class Dashboard extends Component {
```

Comment:
can we rewrite it into functional component as most of our components are?

Reply:
Done 👍